DOC-9810 Should not use 'dpkg -i' to install couchbase-release package

RayOffiah opened the pull request:
Fixed

RayOffiah committed:
DOC-9810 Should not use 'dpkg -i' to install couchbase-release package --> fixed

ceejatec commented on modules/install/pages/ubuntu-debian-install.adoc:
```diff
@@ -161,7 +161,7 @@ sudo apt-get update
+sudo apt-get install -y ./[.var]_package-name_.deb
 ----
 +
 If `dpkg` reports any errors about missing dependencies, issue the following command to download and install those dependencies from the internet:
```
This line and the following [source] block should be removed, since the instructions no longer say to invoke "dpkg". (apt-get will automatically download and install the required dependencies; that's the main reason I recommend using it instead of raw dpkg.)

ceejatec reviewed (changes requested):
(See code comment)

RayOffiah committed:
Removed hang-over text from ubuntu-debian-install.adoc.
Changed installation instructions in install-uninstalling.adoc to use apt to remove the package.

RayOffiah commented on modules/install/pages/install-uninstalling.adoc:
```diff
@@ -50,7 +50,7 @@ Ubuntu and Debian::
 [source,console]
 ----
-sudo dpkg -r couchbase-server
+sudo apt purge couchbase-server
```
@ceejatec <-- Not that familiar with Linux, so is this the correct way to remove the package, given that the intro in the instructions says that all the configuration files have to be removed as well?

RichardSmedley replied:
Yes, that's fine - apt remove takes out an installed package, but apt purge will remove associated config files.

RayOffiah commented:
@ceejatec <-- Fixes done.

RichardSmedley commented:
We should update the version of Couchbase installed in the doc from 6.x to 7.x, on lines 82, 96, 120, and 134.

RayOffiah committed:
Updated version numbers.

RayOffiah committed:
Corrected number

RayOffiah commented:
> We should update the version of Couchbase installed in the doc from 6.x to 7.x, on lines 82, 96, 120, and 134.

@RichardSmedley <-- Done

RichardSmedley reviewed (approved):
LGTM  :+1:
